### DISCUSSION TITLE
出现了关联数据时闪退的bug

### DISCUSSION THREAD

**Discussion by IyoJimbo** [chinosk6/gakuen-imas-localify#30] [General]
安装后  点击关联数据会闪退   设备是小米14

**Comment by chinosk6** [comment]
在哪个界面点击的？具体操作流程是？

**Reply by IyoJimbo** [reply]
tap to start那个主界面   点击数据关联出现了闪退

**Reply by chinosk6** [reply]
使用 LSPatch 还是 LSPosed?

**Reply by IyoJimbo** [reply]
lspatch   使用了对安装包修补  和对应用修补  都出现了这个问题  ;好像不止是点数据关联  点菜单也会闪退  我尝试一下不关联直接进入

**Reply by chinosk6** [reply]
本地测试，无法复现此问题

**Reply by chinosk6** [reply]
有电脑的话，可以使用 [pidcat](https://github.com/JakeWharton/pidcat) 反馈日志。没有 Python 环境的话可以在群里下载 exe。
使用下面的指令查看日志
```sh
pidcat com.bandainamcoent.idolmaster_gakuen
```

**Reply by IyoJimbo** [reply]
在多次尝试后终于成功了
